P5/Source/Specs/move.xml

```diff
@@ -203,7 +134,7 @@ $Id$
         </castList>
       </performance>
       <!-- ... -->
-      <stage type="entrance"><move who="#zh-tw_bellaf" type="enter" where="L" perf="#perf1"/>
+      <stage type="entrance"><move who="#zh-tw_bellaf" type="enter" where="#L" perf="#perf1"/>
```

Comment:
see above for resolving the `where="#L"`. 
And it just occurred to me, that this should probably be 'localized' as well, i.e. `where="#zh-tw_L"` – which is also true for the `@perf` here!